clang-tools-extra/clang-doc/assets/index.js

```diff
@@ -80,8 +80,6 @@ function createIndex(Index) {
 
 // Runs after DOM loads
 document.addEventListener("DOMContentLoaded", function() {
-  // JsonIndex is a variable from another file that contains the index
-  // in JSON format
-  var Index = JSON.parse(JsonIndex);
-  createIndex(Index);
+  // Loadindex is an asynchronous function that will be generated clang-doc
```

Comment:
```suggestion
  // LoadIndex is an asynchronous function that will be generated by clang-doc
```
Maybe this? Otherwise, I'm not sure what's going on, and you'll need to describe it more fully.